src/internal/m365/collection/drive/delta_tree_test.go

```diff
@@ -453,11 +477,15 @@ func (ts tombs) compare(
 	require.Len(t, tombstones, len(ts), "count of tombstoned nodes")
 
 	for _, entombed := range ts {
-		zombey := tombstones[entombed.self.id]
+		expectID := ptr.Val(entombed.self.folder.GetId())
+
+		zombey := tombstones[expectID]
 		require.NotNil(t, zombey, "tombstone must exist")
 		assert.Nil(t, zombey.parent, "tombstoned node should not have a parent reference")
 
-		t.Run("assert_tombstones/"+zombey.id, func(_t *testing.T) {
+		resultID := ptr.Val(zombey.folder.GetId())
+
+		t.Run("assert_tombstones/"+resultID, func(_t *testing.T) {
```

Comment:
nit: I think something besides `/` in the test name would be better. It'll probably cause odd escaping errors when trying to run individual tests